jina/drivers/control.py

```diff
@@ -86,9 +86,8 @@ def __call__(self, *args, **kwargs):
         elif self.req.command == 'CANCEL':
             pass
         elif self.req.command == 'DUMP':
-            self.logger.warning(
-                'DUMP command cannot be processed by ControlReqDriver. Use DumpDriver'
-            )
+            print(f'### DUMP in Driver')
+            self._dump()
```

Comment:
The ControlDriver will all have dump?

Reply:
Yes. We are now just sending a control req to the Pod/Peas, and it will be received by the ControlDriver